lib/mongo/srv/monitor.rb

```diff
@@ -0,0 +1,92 @@
+# Copyright (C) 2014-2019 MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Mongo
+  module SRV
```

Comment:
Since we decided to camel case module names like this, `Srv`?

Reply:
This would be inconsistent with existing user-facing APIs, which we can't change without breaking backwards compatibility: https://github.com/mongodb/mongo-ruby-driver/blob/e3d399d758cc8fa780c02c49c66b8cf4f9700bd6/lib/mongo/error/no_srv_records.rb#L24